Misc/STAR_Jersey Approach AoR.txt

```diff
@@ -0,0 +1,3 @@
+Jersey Approach AoR 	N049.22.07.000 W003.00.00.000 N049.22.08.000 W002.29.51.000
+			N049.22.08.000 W002.29.51.000 N049.42.31.930 W002.11.59.180
+			N049.42.31.930 W002.11.59.180 N049.42.31.930 W002.00.00.000
```

Comment:
```suggestion
Jersey Approach AoR     N049.22.07.000 W003.00.00.000 N049.22.08.000 W002.29.51.000
                        N049.22.08.000 W002.29.51.000 N049.42.31.930 W002.11.59.180
                        N049.42.31.930 W002.11.59.180 N049.42.31.930 W002.00.00.000
```
Minor change here to spaces instead of tabs - this ensures it displays the same across all text editors.